languages/elixir/exercises/concept/numbers/.docs/after.md

```diff
@@ -0,0 +1,10 @@
+One of the key aspects of working with numbers in Elixir is the distinction between integers (numbers without a decimal separator) and floating-point numbers (numbers with one or more digits after a decimal separator).
+
+The two most commonly used numeric types in Elixir are `integer` (a 32-bit integer) and `float` (a 64-bit floating-point number).
```

Comment:
Elixir ints can actually grow beyond 2^31 - 1. In other words they aren't limited to 32 bits.

Reply:
right, they are implemented with arbitrary precision.  Will change

Reply:
```suggestion
The two most commonly used numeric types in Elixir are `integer` (an infinite precision number) and `float` (a 64-bit floating-point number).
```